a_tl.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type HttpResponse struct {
+	url      string
+	response *http.Response
+	err      error
+	req      string
+}
+
+func asyncHttpGets(url string) []*HttpResponse {
+	ch := make(chan *HttpResponse)
+	responses := []*HttpResponse{}
+	client := http.Client{}
+
+	letters := "abcdefghijklmnopqrstuvwxyz0123456789"
+	for _, l := range letters {
+		for _, m := range letters {
+			go func(l, m rune) {
+				fmt.Printf("trying %s\n", string(l)+string(m))
+				// fmt.Printf("Fetching %s \n", url+string(l)+string(m))
+				resp, err := client.Get(url + string(l) + string(m))
+				ch <- &HttpResponse{url, resp, err, string(l) + string(m)}
+				if err != nil && resp != nil && resp.StatusCode != http.StatusOK {
+					fmt.Printf("%s: %d\n", string(l)+string(m), resp.StatusCode)
+				} else {
+					fmt.Printf("* ")
+					resp.Body.Close()
+				}
+			}(l, m)
+			time.Sleep(10 * time.Millisecond)
```

Comment:
Why sleep and why 10ms? Prefer passing delay in parameter. Add a comment on how/why.